### DISCUSSION TITLE
TimeScaleOptions:`rightOffset` with `fixRightEdge`

### DISCUSSION THREAD

**Discussion by appsunited** [tradingview/lightweight-charts#1227] [Q&A]
Hello! 👋 First of all: Thanks for this awesome chart library! 🚀 🤩 

Have a candlestick chart and want some space between the last right candle and the vertical price axis. The `rightOffset` will make it. Also I would like to fix the chart to the left and right edge to avoid scaling the time to much. But if `fixRightEdge` is applied, the `rightOffset` is ignored. 

This is how it should look like before scaling (with `rightOffset`):
<img width="1076" alt="before-scaling" src="https://user-images.githubusercontent.com/501784/208189518-404e64e4-c100-4e58-b5a2-cbd41a3f0a93.png">

This is scaling too much (neither `fixRightEdge` nor `fixLeftEdge`):
<img width="1079" alt="scaled-too-far" src="https://user-images.githubusercontent.com/501784/208190290-630efba6-796f-4a86-9079-f82284d85a9a.png">

Scales too much to the left edge (with `fixLeftEdge`):
<img width="1076" alt="scaled-to-the-left-edge" src="https://user-images.githubusercontent.com/501784/208190458-87b600bd-e680-47a6-ba90-d87e4e40671f.png">

This prevents scaling too much, but `rightOffset` is ignored due the applied `fixRightEdge` and `fixLeftEdge`:
<img width="1081" alt="fixed-to-both-edges-missing-offset" src="https://user-images.githubusercontent.com/501784/208190684-5aef0b60-0433-4427-a904-aed3e639ba0d.png">

Is there a way to achieve a result like `rightOffset` and `fixRightEdge`together?

**Comment by michal-th** [comment]
I'm facing the same issue, did anyone find a solution to this?

**Reply by appsunited** [reply]
Unfortunately I did not find a solution so far. 😕
